xml/ay_bigfile.xml

```diff
@@ -1029,6 +1029,37 @@ exit 0
       <row>
        <entry>
         <para>
+         <literal>self_update</literal>
+        </para>
+       </entry>
+       <entry>
+        <para>
+         This option allows to enable/disable the self-update.
+         Check out the &deploy; to find further information about this feature.
```

Comment:
Please replace `&deploy;` with the following:

    <xref linkend="sec.i.yast2.self_update"/>

The will create a link to the respective section in the deployment guide.